model/variableSheet.cc

```diff
@@ -53,8 +53,9 @@ namespace
 void VariableSheet::populateItemVector() {
   itemVector.clear();	
   minsky().canvas.model->recursiveDo(&GroupItems::items,
-                                        [&](Items&, Items::iterator i) {                                 
-                                          itemVector.emplace_back(*i);
+                                        [&](Items&, Items::iterator i) {
+									      if ((*i)->variableCast()->type()!=VariableType::parameter)		                                 
```

Comment:
Don't you want to reverse the logic here, ie (*i)->variableCast()->type()==VariableType::parameter

Reply:
I thought we wanted separate parameter and variable tabs, because we want to make the former editable, while flow vars, in particular, shouldn't be?